packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -5,31 +5,38 @@
          @drop.prevent="handleDrop">
 
         <VerificationDropBox
+            v-if="!digitalTwin.processing"
```

Comment:
is there a difference between checking here for `digitalTwin.processing` instead for `digitalTwin.active`?

Reply:
Removed it, it was just for testing

Reply:
we still need `v-if="!digitalTwinMode"` right? Or do we want to show the verification drop box on dragging files in digital twin mode also?

Reply:
btw, when i open http://localhost:8080/?file=https://api.dev.testnet.certifaction.io/storage/0xfd7367405ceba3d627753868eca7c1db9dc4825489be5c4e5e54d6693840b11b#e8066280098a352785f0c7fceed0768d0bba7bdefcce21e1f9cb382e7452ba12. It always shows first the normal verification for around 1 second, like a flickering, and then switches to digital twin view. Is it possible to prevent that?

Reply:
`digitalTwinMode` I'd rename to either `isDigitalTwinMode` or `digitalTwinModeActive`, with the name now was expecting a string that tells me the mode

Reply:
No you are right, added the condition now to VerificationDropBox as well. Renamed it to digitalTwinModeActive. Yes i saw the flickering now also while implementing the viewer, before i didn't have this issue. Will handle it in the viewer branch and add more polished transitions.

Reply:
👌